### ISSUE TITLE
Validate from stdin

### ISSUE THREAD

**Issue by alexzielenski** [kubernetes-sigs/kubectl-validate#32]
I think it'd be nice to support stdin as an input source. Not sure what the arguments would look like. We could provide `-` similarly to kubectl to indicate files from std in should be validated too?

**Comment by HirazawaUi** [comment]
/assign 
I would like work for this

**Comment by HirazawaUi** [comment]
I agree that using the same way as kubectl to receive input from std would make it less confusing and easier for administrators using kubernetes to get started

**Comment by HirazawaUi** [comment]
Do we need to continue the discussion on this issue? If not, maybe it's time to get into coding

**Comment by HirazawaUi** [comment]
@lauchokyip  I'd like to get some responses before I start coding, so I've been waiting, but it's probably good for now too, you can continue to track this issue
/unassign 
/assign lauchokyip

**Comment by MPV** [comment]
May we please have some more eyes on the PR that solves this now?
- https://github.com/kubernetes-sigs/kubectl-validate/pull/125
